### PR TITLE
feat: Complete the implementation of the  latest APIs.

### PR DESCRIPTION
**Pull request by AsakusaRinne**
Sync with the latest APIs of ```Casbin.NET 2.x```.

### CODE REVIEW

**Comment by hsluoyz** [comment]
@sagilio plz review

File: Casbin.Adapter.EFCore/Extensions/CasbinRuleExtenstion.cs [hunk 1]
```diff
@@ -39,6 +39,30 @@ internal static List<string> ToList(this ICasbinRule rule)
             return list;
         }
 
+        internal static void SetValue(this ICasbinRule rule, IPolicyValues values)
```
Comment (sagilio) [inline Casbin.Adapter.EFCore/Extensions/CasbinRuleExtenstion.cs:42]:
```suggestion
        internal static void SetValues(this ICasbinRule rule, IPolicyValues values)
```

File: Casbin.Adapter.EFCore/Extensions/CasbinRuleExtenstion.cs [hunk 2]
```diff
@@ -188,44 +212,82 @@ internal static IQueryable<TCasbinRule> ApplyQueryFilter<TCasbinRule>(this IQuer
...
+
+            if (count > 5)
+            {
+                rule.V5 = ruleStrings[5];
+            }
+
+            return rule;
+        }
+
+        internal static TCasbinRule Parse<TCasbinRule>(string policyType, IPolicyValues ruleStrings)
```
Comment (sagilio) [inline Casbin.Adapter.EFCore/Extensions/CasbinRuleExtenstion.cs:254]:
```suggestion
        internal static TCasbinRule Parse<TCasbinRule>(string policyType, IPolicyValues values)
```

File: Casbin.Adapter.EFCore/EFCoreAdapter.cs [hunk 3]
```diff
@@ -295,6 +295,66 @@ public virtual async Task RemovePoliciesAsync(string section, string policyType,
...
+            IReadOnlyList<IPolicyValues> oldRules, IReadOnlyList<IPolicyValues> newRules)
+        {
+            if (oldRules is null || newRules is null || newRules.Count  != oldRules.Count)
+            {
+                return;
+            }
+
+            for (int i = 0; i < oldRules.Count; i++)
+            {
+                UpdatePolicyAsync(section, policyType, oldRules[i], newRules[i]);
```
Comment (sagilio) [inline Casbin.Adapter.EFCore/EFCoreAdapter.cs:352]:
```suggestion
                await UpdatePolicyAsync(section, policyType, oldRules[i], newRules[i]);
```

**Review by sagilio — CHANGES_REQUESTED** [review]
Most LGTM

**Comment by hsluoyz** [comment]
@sagilio

**Review by sagilio — APPROVED** [review]
LGTM

**Comment by sagilio** [comment]
@AsakusaRinne plz sign off the commits 
image

**Comment by AsakusaRinne** [comment]
> @AsakusaRinne plz sign off the commits image

OK, done.

**Pull request merged**